packages/angular_devkit/build_angular/test/hello-world-app/tsconfig.json

```diff
@@ -8,12 +8,10 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "skipLibCheck": true,
```

Comment:
@alan-agius4 heads-up for this one: I had to disable it, because there seems to be some sort of mismatch between the types in the Bazel bin and what is used in the rest of the project.